data/trie/export_test.go

```diff
@@ -0,0 +1,23 @@
+package trie
+
+func (ts *trieSyncer) trieNodeIntercepted(hash []byte, val interface{}) {
+	ts.mutOperation.Lock()
+	defer ts.mutOperation.Unlock()
+
+	log.Trace("trie node intercepted", "hash", hash)
+
+	n, ok := ts.nodesForTrie[string(hash)]
+	if !ok || n.received {
```

Comment:
This condition is ok?

Reply:
it was moved from the sync.go file. Used only in unit tests. Guess it is ok as the node is ignored if it does not exists or already received

Reply:
ok